### PR TITLE
(RE-12395) Release 0.99.35 prep

### PR DESCRIPTION
**Pull request by e-gris**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.99.35] - 2019-06-11
+### Fixed
+- Don't use `set -e` for windows signing. This will let retries work.
+### Changed
+- Use HEREDOC for msi sign command
+
```
Comment (underscorgan) [inline CHANGELOG.md]: There's also a link at the bottom of this file that needs updating

Comment (e-gris) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.99.35] - 2019-06-11
+### Fixed
+- Don't use `set -e` for windows signing. This will let retries work.
+### Changed
+- Use HEREDOC for msi sign command
```
Comment (mwaggett) [inline CHANGELOG.md]: This is minimal enough that I don't think it needs to go into the changelog, but I won't block on that.

Comment (mwaggett) [reply CHANGELOG.md]: minimal is the wrong work - this isn't a user-facing change, *that's* why I don't think it needs to go in the changelog.

**Pull request merged**